add script_runner method in fuels-test-helpers (#325)
FuelLabs/fuels-rs: docs/src/testing/run-script.md

# run_script 

`run_script` is helper function for testing simple Sway scripts and reducing boilerplate code related to setting up contracts and deployment.
It takes the path to the generated `.bin` file as argument.

You can use it this way:

````rust

> #[tokio::test]
> async fn test_logging_sway() {
> 
>     let path_to_bin = "tests/test_projects/logging/out/debug/logging.bin";
>     let return_val = run_script(path_to_bin).await;
> 
>     let correct_hex =
>         hex::decode("ef86afa9696cf0dc6385e2c407a6e159a1103cefb7e2ae0636fb33d3cb2a9e4a");
> 
>     assert_eq!(correct_hex.unwrap(), return_val[0].data().unwrap());
> 
> }

Reviewer: ```suggestion
{{#include ../../../packages/fuels-test-helpers/src/script.rs:test_logging_sway}}
```

# `run_script` 

> `run_script` is helper function for testing simple sway code and reducing boilerplate code in tests.

Reviewer 2: ```suggestion
`run_script` is helper function for testing simple Sway scripts and reducing boilerplate code related to setting up contracts and deployment.
```

> As a parameter the function receives a `&str`, that is, the path that leads to the generated `.bin` file.

Reviewer 2: ```suggestion
It takes the path to the generated `.bin` file as argument.
```

> 
> ## Usage
> 
> A simple example of testing using `run_script`
> 

Reviewer 2: ```suggestion
You can use it this way:

```
